### PR TITLE
LRDOCS-4769 oauth2 review

### PR DESCRIPTION
**Pull request by stian-sigvartsen**
/cc @csierra @topolik

### CODE REVIEW

**Comment by stian-sigvartsen** [comment]
Hi @sez11a ! This is a great start! I've fleshed it out a bit, especially around the grant flows and how access tokens are obtained.

This content looks good for developer documentation. We also need portal administrator documentation.

I've CC'ed Carlos who will contributing more content.

File: discover/deployment/articles/04-securing-liferay/15-oauth2/02-authorizing-access.markdown [hunk 1]
```diff
@@ -12,45 +12,89 @@ applications. The URL for this requires the following request parameters:
 
 - `response_type` 
 - `client_id` 
-- `client_secret` (optional)
+- `client_secret`
 
 To construct a URL for this authorization, therefore, follow this pattern: 
 
-    https://[hostname]/o/oauth2/authorize?response_type=code&client_id=[client ID]
+    https://[hostname]/o/oauth2/authorize?response_type=code&client_id=[client ID]&client_secret=[client secret]
```
Comment (csierra) [inline discover/deployment/articles/04-securing-liferay/15-oauth2/02-authorizing-access.markdown]: @stian-sigvartsen ... is the client_secret mandatory in this request? I would say it SHOULD NOT go in this request, should it?

Comment (stian-sigvartsen) [reply discover/deployment/articles/04-securing-liferay/15-oauth2/02-authorizing-access.markdown]: Sorry @sez11a , Carlos is right. I'll amend

**Comment by sez11a** [comment]
Merged and will be published this week. Thank you!

**Pull request closed without merging**